students/joejohnsto/lesson03/mailroom.py

```diff
@@ -7,40 +7,99 @@
 Mailroom Part I
 """
 
+import sys
 
 donor_db = [('Bill Gates', [4000.32, 35.00, 17899.99]),
             ('Oprah Winfrey', [9999.99, 9999.99]),
             ('Rob Schneider', [400.00, 55.00, 800.00]),
             ('Donald Trump', [0.32, 5.00]),
             ('Denise Richards', [6040.77])]
 
-prompt = "\n".join(("Welcome to the mailroom!",
+prompt = "\n".join(("\nWelcome to the mailroom!",
           "Please choose from below options:",
           "1 - Send a Thank You",
           "2 - Create a Report",
           "3 - quit",
           ">>> "))
 
 
+def sort_key(donor):
```

Comment:
For this kind of "micro-functions" you could use lambda functions.

Reply:
are we going to learn lambda functions in class at some point?